app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
we have a bit of confusion here, since there are server sent validation messages, but also html5 popups (in case you didnt' enter a @ in the mail address).

@jimmykarily can we leave it now as it is and follow up on this in another PR? I'll create an issue. i know it's bad to have no notifications now, but i don't want to mess around more with the UI and how to present the errors in this PR. i think it deserves some contained effort in a separate PR.